.github/workflows/autobuild.yml

```diff
@@ -14,21 +13,19 @@
 #     for every tag that starts with 'r' and does not have any suffix:
 #          - do CodeQl while building for every platform
 #          - publish the created binaries/packs only as artifacts/appendix as a release
-#
-
 
 on:
   workflow_dispatch:
   push:
     tags:
       - "r*"
     branches:
-      - "autobuild*"  # for developers: branches starting with autobuild will be built and evaluated on each push
-      - "autobuild/*" # for developers: branches starting with autobuild will be built and evaluated on each push
+      # For developers: Branches starting with autobuild will be built and evaluated on each push.
+      - "autobuild**"
       - "master"
-    #  - "*"
-  pull_request: # The branches below must be a subset of the branches in "push"
+  pull_request:
     branches:
+      # These branches must be a subset of the branches in "push":
```

Comment:
```suggestion
      # Only automatically run the build on pushes to master to avoid wasting ressources
```

Reply:
Background: https://github.com/jamulussoftware/jamulus/pull/978#issuecomment-778540855

So... now I understand what was probably meant by the original comment. But rather than explaining Why (CodeQL mandates that) it explains the What (the fact that this list is a subset of the other can be derived without the comment). I've now added a comment to explain the reasoning. I've also added the above suggestion.

Reply:
Yes. That's good.